sherpa/astro/tests/test_astro.py

```diff
@@ -293,9 +293,9 @@ def test_radpro_dm(self):
 
             fres = ui.get_fit_results()
             covarerr = sqrt(fres.extra_output['covar'].diagonal())
-            assert covarerr[0] == approx(6.98861, rel=1e-4)
-            assert covarerr[1] == approx(0.383065, rel=1e-4)
-            assert covarerr[2] == approx(0.101712, rel=1e-4)
+            self.assertEqualWithinTol(covarerr[0], 9.37344776, 1e-4)
```

Comment:
A question for @olaurino - should we try and stick with the use of `pytest.approx` here, rather than the `assertEqualWithinTol` method? I would like to be able to remove the `SherpaTestCase` class (or whatever it is called) in the long term, *but* I haven't looked at the meaning of the tolerances in the two paths (e.g. does `pytest.approx` only provide relative, absolute, or both tolerances, and how does it map to the third parameter of `assertEqualWithinTol`).

Reply:
Hmm... I thought I had already replied to this. We should be consistent in the use of `pytest.approx`.